library/core/src/panic.rs

```diff
@@ -5,6 +5,40 @@
 use crate::any::Any;
 use crate::fmt;
 
+#[doc(hidden)]
+#[unstable(feature = "edition_panic", issue = "none", reason = "use panic!() instead")]
```

Comment:
You could mention the tracking issue #80162 here.

Reply:
This is never going to be stable. It is an internal implementation detail.